src/OpenLoco/src/Gui.cpp

```diff
@@ -20,11 +20,12 @@ namespace OpenLoco::Gui
     {
         Windows::Main::open();
 
-        addr<0x00F2533F, int8_t>() = 0; // grid lines
-        addr<0x0112C2e1, int8_t>() = 0;
-        addr<0x009c870E, int8_t>() = 1;
-        addr<0x009c870F, int8_t>() = 2;
-        addr<0x009c8710, int8_t>() = 1;
+        Windows::Main::resetGridlines();
+        Windows::Main::resetDirectionArrows();
+
+        Windows::Terraform::setAdjustLandToolSize(1);
```

Comment:
NB: this was being reset to 0, but that was an oversight on my part. The mountain tool is now a different mode, which means 0 no longer carries meaning for the land tool.